Library/Homebrew/utils/puts_columns.rb

```diff
@@ -0,0 +1,41 @@
+require "utils/tty"
+
+module Kernel
```

Comment:
I see. Personally I'd rather we had a class method that took an argument and move away from monkeypatching system classes.